components/sensor/mcp9808.rst

```diff
@@ -0,0 +1,55 @@
+MCP9808 Temperature Sensor
+==========================
+
+.. seo::
+    :description: Instructions for setting up MCP9808 temperature sensor.
+    :image: mcp9808.jpg
+    :keywords: MCP9808
+
+The MCP9808 Temperature sensor allows you to use your MCP9808
+(`datasheet <http://ww1.microchip.com/downloads/en/DeviceDoc/25095A.pdf>`__,
+`Adafruit <https://www.adafruit.com/product/1782>`__) sensors with
+ESPHome. The :ref:`I²C Bus <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/mcp9808-full.jpg
+    :align: center
+    :target: `Adafruit`_
+    :width: 50.0%
+
+    MCP9808 Temperature Sensor. Image by `Adafruit`_.
+
+.. _Adafruit: https://www.adafruit.com/product/1782
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: mcp9808
+        name: "Living Room Temperature"
+        update_interval: 60s
+
+Configuration variables:
+------------------------
+
+- **temperature** (**Required**): The information for the temperature sensor.
+
+  - **name** (**Required**, string): The name for the temperature sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
```

Comment:
Please also add `- **address** ...` as in other i2c-based sensors (this config option is automatically inserted by `i2c.i2c_device_schema`)

Reply:
I think I got this added, thanks for pointing it out!